Application/Education/Certificate/Setting/Frontend.php

```diff
@@ -194,7 +282,8 @@ private function getSubject(
         $LaneIndex,
         $LaneRanking,
         $LaneTitle = '',
-        $FieldName = 'Subject'
+        $FieldName = 'Subject',
+        $PreFach = ''
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$PreFach" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)